OpenRA.Mods.Common/Widgets/Logic/Settings/GamePlaySettingsLogic.cs

```diff
@@ -0,0 +1,127 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.Common.Widgets.Logic
+{
+	public class GameplaySettingsLogic : ChromeLogic
+	{
+		[FluentReference]
+		const string AutoSaveIntervalOptions = "auto-save-interval.options";
+
+		[FluentReference]
+		const string AutoSaveIntervalDisabled = "auto-save-interval.disabled";
+
+		[FluentReference]
+		const string AutoSaveIntervalMinuteOptions = "auto-save-interval.minute-options";
+
+		[FluentReference]
+		const string AutoSaveMaxFileNumber = "auto-save-max-file-number";
+		readonly int[] autoSaveSeconds = { 0, 10, 30, 45, 60, 120, 180, 300, 600 };
```

Comment:
Will game save be skipped if on pause? Or will that automatically work as expected since you use game ticks and not time.

Reply:
ticks don't tick when the game is paused